test/src/main/java/org/apache/accumulo/test/upgrade/UpgradeIT.java

```diff
@@ -139,14 +167,30 @@ public void testUpgradeManagerFailure() throws Exception {
     ServerContext ctx = getCluster().getServerContext();
     UpgradeUtilIT.downgradePersistentVersion(ctx);
 
-    List<String> args = new ArrayList<>();
-    args.add("--props");
-    args.add(getCluster().getAccumuloPropertiesPath());
-    try (TestManager mgr = new TestManager(args.toArray(new String[0]))) {
-      IllegalStateException ise = assertThrows(IllegalStateException.class, () -> mgr.runServer());
-      assertTrue(ise.getMessage().equals("Upgrade not started, " + Constants.ZUPGRADE_PROGRESS
-          + " node does not exist. Did you run 'accumulo upgrade --start'?"));
-    }
+    final AtomicReference<Throwable> err = new AtomicReference<>();
+    Thread testMgrThread = new Thread() {
+      @Override
+      public void run() {
+        try {
+          List<String> args = new ArrayList<>();
+          args.add("--props");
+          args.add(getCluster().getAccumuloPropertiesPath());
+          TestManager mgr = new TestManager(args.toArray(new String[0]));
+          mgr.runServer();
+          mgr.close();
```

Comment:
mgr.close() will never be called when an excepting is thrown from mgr.runServer().  Tried changing to the following the test fails w/ the process halting.  Calling TestManager.close() seems to halt the test process.

```java
         try(TestManager mgr = new TestManager(args.toArray(new String[0]))) {
           mgr.runServer();
         }
```

Reply:
We can remove the call to `mgr.close()` in these changes.